code/modules/mob/living/carbon/human/dummy.dm

```diff
@@ -79,6 +79,8 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
 	randomize_human(src)
 	dna.initialize_dna(skip_index = TRUE) //Skip stuff that requires full round init.
 
+/mob/living/carbon/human/dummy/log_mob_tag(text)
```

Comment:
Empty functions should include `return`.

Reply:
Done, is this just for readability or Byond related?

Reply:
Readability